File: app/src/main/java/com/saitejajanjirala/imdbclone/listeners/listenerobject/BookMarkListenerObject.java
package com.saitejajanjirala.imdbclone.listeners.listenerobject;

import com.saitejajanjirala.imdbclone.listeners.BookMarkListener;
import com.saitejajanjirala.imdbclone.listeners.BookMarkStatusesListener;

import java.io.Serializable;

public class BookMarkListenerObject implements Serializable {
    public BookMarkStatusesListener listener;

    public BookMarkListenerObject(BookMarkStatusesListener listener) {
        this.listener = listener;
    }
}

File: app/src/main/java/com/saitejajanjirala/imdbclonekotlin/listeners/listenerobject/BookMarkListenerObject.kt
package com.saitejajanjirala.imdbclonekotlin.listeners.listenerobject

import com.saitejajanjirala.imdbclonekotlin.listeners.BookMarkStatusesListener
import java.io.Serializable

class BookMarkListenerObject(listener: BookMarkStatusesListener) : Serializable {
    var listener: BookMarkStatusesListener = listener

}